types/method-override/package.json

```diff
@@ -1,11 +1,11 @@
 {
     "private": true,
     "name": "@types/method-override",
-    "version": "0.0.9999",
+    "version": "3.0.9999",
     "projects": [
         "https://github.com/expressjs/method-override"
     ],
-    "dependencies": {
+    "peerDependencies": {
         "@types/express": "*"
     },
```

Comment:
Actually, looking at https://www.npmjs.com/package/method-override, this package doesn't even reference `express` at all. It's not a strict requirement that `express` be installed for the package to be useful. Is this actually accurate? Anyone who has this will now be forced to install the express types manually.

(It is an express project, so maybe that is okay? Not sure how to get a read on how many `package.json`s with `@types/method-override` do/don't also have `@types/express`)

Reply:
I'm not an express expert, but it seems pretty clear to me that this package is primarily intended to be used as a middleware based on the shape of its export. Since the types intentionally augment the express types, I'm not sure what the alternative would be?

Reply:
Yeah, I don't think there really is an alternative. If the first parameter were required and didn't include `string |`, then there would be a generic constraint that could return something, but, no dice.

Reply:
So, I guess we'll just try this; it's at least a major bump and we can always undo it. It's unfortunate that regular `*` deps work so poorly.